Gemfile.jruby-1.9.lock

```diff
@@ -167,16 +167,16 @@ GEM
       i18n (= 0.6.9)
       jrjackson (~> 0.3.7)
       jruby-openssl (= 0.9.13)
-      logstash-core-event (= 2.3.1.snapshot1)
+      logstash-core-event (= 2.3.1)
```

Comment:
Actually bundler will use the platform to fetch the correct gems, there is never the `-java` in the name of the gems. See https://rubygems.org/gems/logstash-core-event, there is a `-java` after the version number. You can release multiples gem with the same name but with different platform target.

 The exception if the `logstash-core-event-java` see https://rubygems.org/gems/logstash-core-event-java